code/modules/antagonists/eldritch_cult/eldritch_structures.dm

```diff
@@ -0,0 +1,161 @@
+/obj/structure/eldritch_crucible
+	name = "Mawed Crucible"
+	desc = "Immortalized cast iron, the steel-like teeth holding it in place, it's vile extract has the power of rebirthing things, remaking them from the very beginning."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "crucible"
+	anchored = FALSE
+	density = TRUE
+	var/current_mass = 5
+	var/max_mass = 5
+	var/in_use = FALSE
+
+/obj/structure/eldritch_crucible/examine(mob/user)
+	. = ..()
+	if(!IS_HERETIC(user) && !IS_HERETIC_MONSTER(user))
+		return
+	if(current_mass < max_mass)
+		. += "The Crucible requires [max_mass - current_mass] more organs or bodyparts!"
+	else
+		. += "The Crucible is ready to be used!"
+
+	. += "You can anchor and reanchor it using Codex Cicatrix!"
+	. += "It is currently [anchored == FALSE ? "unanchored" : "anchored"]"
+
+/obj/structure/eldritch_crucible/attacked_by(obj/item/I, mob/living/user)
+	if(istype(I,/obj/item/nullrod))
+		qdel(src)
+		return
+
+	if(!IS_HERETIC(user) && !IS_HERETIC_MONSTER(user))
+		if(iscarbon(user))
+			devour(user)
+		return
+
+	if(istype(I,/obj/item/forbidden_book))
+		playsound(src, 'sound/misc/desecration-02.ogg', 75, TRUE)
+		anchored = !anchored
+		to_chat(user,"<span class='notice'>You [anchored == FALSE ? "unanchor" : "anchor"] the crucible</span>")
+		return
+
+	if(istype(I,/obj/item/bodypart) || istype(I,/obj/item/organ))
```

Comment:
are robotic organs, prosthetic limbs, and flashlight eyes meant to count for this?

Reply:
fixed

Reply:
damn